cybox/common/vocabs.py

```diff
@@ -6,13 +6,58 @@
 from cybox.common import PatternFieldGroup
 from cybox.utils import normalize_to_xml, denormalize_from_xml
 
-INVALID_XSI_TYPE = "bad_xsi_type"
+
+class VocabField(cybox.TypedField):
```

Comment:
@gtback, yup! It's slightly different than the way `TypedField` works, which forces input into a `type_` object. We could certainly modify `TypedField` to mimic the behavior of `VocabField` by introducing a new arg, though right now I think it's only really needed for `VocabString` fields, so maybe it makes sense to leave it as a subclassed approach?